core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -78,7 +78,8 @@ paths:
       tags:
         - billing
       summary: create billing project in rawls and google
-      description: create new billing project in rawls and google
+      description: Create new billing project in rawls and google. Deprecated, use v2 version.
+      deprecated: true
```

Comment:
Deprecated everything except listBillingAccounts, which does not  have a v2 version.

![image](https://user-images.githubusercontent.com/484484/191557061-af1128fd-a9a9-4bd9-80ec-f52b284299d8.png)